MaxMind.Db/MemoryMapBuffer.cs

```diff
@@ -55,15 +64,26 @@ public override byte[] Read(long offset, int count)
 
         public override byte ReadOne(long offset) => _view.ReadByte(offset);
 
-        public override string ReadString(long offset, int count) => Encoding.UTF8.GetString(Read(offset, count));
+        public override string ReadString(long offset, int count)
+        {
+            var buffer = Read(offset, count);
+            return Encoding.UTF8.GetString(buffer, 0, buffer.Length);
+        }
 
         public override void Copy(long offset, byte[] bytes)
         {
             // Although not explicitly marked as thread safe, from
             // reviewing the source code, these operations appear to
             // be thread safe as long as only read operations are
             // being done.
+#if !NETSTANDARD1_4
```

Comment:
I see you opened [this issue](https://github.com/dotnet/coreclr/issues/4799). Thanks! Could you add a link to it so that we can update the code if it gets resolved?